generate-protos.sh

```diff
@@ -22,27 +26,27 @@ case "$OSTYPE" in
     exit 1
 esac
 
-echo "Cloning google-cloudevents"
+echo "- Cloning github.com/googleapis/google-cloudevents into tmp"
 # For the moment, just clone google-cloudevents. Later we might make
 # it a submodule. We clone quietly, and only with a depth of 1
 # as we don't need history.
 rm -rf tmp
-mkdir tmp
+mkdir -p tmp
```

Comment:
There's no need for that - the parent directory of `tmp` clearly exists as it's the current directory